lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -293,11 +298,30 @@ sub log_entry {
     );
 }
 
-sub _stderr_is_tty{
+=head2 _process_data
+
+Process the data before printing out.
+
+Parameters: $self, $data
+Return: processed data
```

Comment:
suggestion use standard format for pod
https://wikijs.deriv.cloud/en/Backend/guides/perl/perl-style-guide Subroutine Comments format.